solutions/observability/streams/management/extract.md

```diff
@@ -51,80 +74,85 @@ Processors support these comparators:
 - ends with
 - exists
 - not exists
+:::
 
 ### Preview changes [streams-preview-changes]
 
-Under **Processors for field extraction**, when you set pipeline processors to modify your documents, **Data preview** shows you a preview of the results with additional filtering options depending on the outcome of the simulation.
+After you create processors, the **Data preview** tab simulates processor results with additional filtering options depending on the outcome of the simulation.
 
 When you add or edit processors, the **Data preview** updates automatically.
 
 :::{note}
-To avoid unexpected results, we recommend adding processors rather than removing or reordering existing processors.
+To avoid unexpected results, it's best to add processors rather than remove or reorder existing ones.
 :::
 
-**Data preview** loads 100 documents from your existing data and runs your changes using them.
-For any newly added processors, this simulation is reliable. You can save individual processors during the preview, and even reorder them.
-Selecting **Save changes** applies your changes to the data stream.
+**Data preview** loads 100 documents from your existing data and runs your changes against them.
+For any newly created processors and conditions, the preview results are reliable, and you can freely create and reorder during the preview.
 
-If you edit the stream again, note the following:
-- Adding more processors to the end of the list will work as expected.
-- Changing existing processors or re-ordering them may cause unexpected results. Because the pipeline may have already processed the documents used for sampling, the UI cannot accurately simulate changes to existing data.
-- Adding a new processor and moving it before an existing processor may cause unexpected results. The UI only simulates the new processor, not the existing ones, so the simulation may not accurately reflect changes to existing data.
+Select **Save changes** to apply your changes to the data stream.
```

Comment:
This sentence almost feels out of place since this section explains how the data preview works.